packages/app-server-issuer/src/exchange-provider/ExchangeProviderManager.ts

```diff
@@ -0,0 +1,167 @@
+import { number, bytes } from '@ckb-lumos/codec';
+import { Cell, helpers, HexString, BI as BigNumber } from '@ckb-lumos/lumos';
+import { AcpTransferSudtBuilder } from '@ckitjs/ckit';
+import { loggerWithModule } from '../logger';
+import { ServerContext } from '../types';
+import { Config } from './config';
+
+export class ExchangeProviderManager {
+  private cells: Cell[] = [];
+  private initiated: boolean;
+  private config!: Config;
+  private context!: ServerContext;
```

Comment:
```suggestion
  private config: Config;
  private context: ServerContext;
```

can we remove `!` here? we can throw an error when constructing failed

Reply:
I made it a singleton, the configs will not initiated in constructor. Just like this.
```typescript
ExchangeProviderManager.getInstance().initiateConfig(config);
```
So that we can call the `getInstance` without any parameters.